graph/src/data/query/error.rs

```diff
@@ -78,10 +93,59 @@ impl fmt::Display for QueryExecutionError {
                 f,
                 "Only a single top-level field is allowed in subscriptions"
             ),
+            QueryExecutionError::SupgraphIdError(s) => {
+                write!(f, "Failed to get subgraph ID from type: {}", s)
+            }
+            QueryExecutionError::RangeArgumentsError(s) => write!(
+                f,
+                "Range inputs must be an integer, {:}",
+                s.iter().fold(String::new(), |acc, value| acc + &value)
+            ),
+            QueryExecutionError::InvalidFilterError => write!(f, "Filter must by an object"),
+            QueryExecutionError::EntityAttributeError(e, a) => {
+                write!(f, "Entity {} has no attribute {}", e, a)
+            }
+            QueryExecutionError::ListTypesError(s) => write!(
+                f,
+                "Values passed to filter {} must be of the same type but are of different types",
+                s
+            ),
+            QueryExecutionError::ListFilterError(s) => {
+                write!(f, "Non-list value passed to {} filter", s)
+            }
+            QueryExecutionError::ValueParseError(t, e) => {
+                write!(f, "Failed to decode {} value, {}", t, e)
+            }
+            QueryExecutionError::AttributeTypeError(value, ty) => write!(
+                f,
+                "Query contains attribute with invalid type, {}:{}",
```

Comment:
`Attributes` are not a thing here. Make this `Query contains value with invalid type {}: {}`. The first is the type, the second is the value, so swap the other arguments to `write!`.